packages/react/src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -211,15 +211,21 @@ const TableToolbar = ({
     className={classnames(`${iotPrefix}--table-toolbar`, className)}
   >
     <TableBatchActions
-      data-testID={`${testID}-batch-actions`}
+      data-testid={`${testID}-batch-actions`}
       className={`${iotPrefix}--table-batch-actions`}
       onCancel={onCancelBatchAction}
       shouldShowBatchActions={hasRowSelection === 'multi' && totalSelected > 0}
       totalSelected={totalSelected}
       translateWithId={(...args) => tableTranslateWithId(i18n, ...args)}
     >
       {batchActions.map(({ id, labelText, ...others }) => (
-        <TableBatchAction key={id} onClick={() => onApplyBatchAction(id)} {...others}>
+        <TableBatchAction
+          key={id}
+          onClick={() => onApplyBatchAction(id)}
+          tabIndex={hasRowSelection === 'multi' && totalSelected > 0 ? 0 : -1}
```

Comment:
We write `hasRowSelection === 'multi' && totalSelected > 0` 3 times here including the prop `shouldShowBatchActions`. Perhaps it is better to place it in a const.

Reply:
good call. I didn't do it initially, since it made the diff look like way more changes than it was. Done now.